src/client.ts

```diff
@@ -3073,4 +3073,54 @@ export class StreamChat<
       ...options,
     });
   }
+
+  /**
+   * _createImport - Create an Import Task.
+   *
+   * Note: Do not use this.
+   * It is present for internal usage only.
+   * This function can, and will, break and/or be removed at any point in time.
+   *
+   * @private
+   * @param {string} filename filename of uploaded data
+   *
+   * @return {APIResponse} An ImportTask
+   */
+  async _createImport(filename: string) {
+    return await this.post<APIResponse & { import_task: {}; upload_url: string }>(this.baseURL + `/imports`, {
+      filename,
+    });
+  }
+
+  /**
+   * _getImport - Get an Import Task.
+   *
+   * Note: Do not use this.
+   * It is present for internal usage only.
+   * This function can, and will, break and/or be removed at any point in time.
+   *
+   * @private
+   * @param {string} id id of Import Task
+   *
+   * @return {APIResponse} An ImportTask
+   */
+  async _getImport(id: string) {
+    return await this.get<APIResponse & { import_task: {} }>(this.baseURL + `/imports/${id}`);
+  }
+
+  /**
+   * _listImports - Lists Import Tasks.
+   *
+   * Note: Do not use this.
+   * It is present for internal usage only.
+   * This function can, and will, break and/or be removed at any point in time.
+   *
+   * @private
+   * @param {{ limit?: number; offset?: number }} options pagination options
+   *
+   * @return {APIResponse} An ImportTask
```

Comment:
We should have a specific response type for this method and for the others as well, right?

Reply:
I'll add it

Reply:
this isn't exactly true since it's a richer version of it 

maybe we should define listResponse, getResponse etc. How is it done in other endpoints?